ElectronApp/src/__tests__/App.test.tsx

```diff
@@ -0,0 +1,51 @@
+import { render, screen } from '@testing-library/react';
+import userEvent from '@testing-library/user-event';
+import { MemoryRouter } from 'react-router-dom';
+
+import '@testing-library/jest-dom';
+
+import App from '../renderer/App';
+import Nav from '../renderer/Nav';
+
+describe('App', () => {
+  it('should render', () => {
+    expect(render(<App />, { wrapper: MemoryRouter })).toBeTruthy();
+  });
+});
+
+describe('Nav', () => {
+  it('should render', () => {
+    expect(render(<Nav />, { wrapper: MemoryRouter })).toBeTruthy();
+  });
+
+  it('should switch to Data Entry', () => {
+    render(<App />, { wrapper: MemoryRouter });
+
+    const leftClick = { button: 0 };
```

Comment:
To reduce the number of times we declare leftClick, I recommend placing this declaration just below the describe.